GoodDollar/GoodDAPP
# (Bug) Fix gundb wrong indexeddb setup

# Description

Correctly initialize rindexeddb

About #974 

To test:
- in devtools open application and browse indexeddb, should not be empty
- gun.get('users/byemail').get(someexistingemailofanotheruser).get('profile').then() - should return all the fields and not just the soul
